src/Time/StepChoosers/StepChooser.hpp

```diff
@@ -18,84 +19,76 @@ namespace Parallel {
 template <typename Metavariables>
 class GlobalCache;
 }  // namespace Parallel
-template <typename StepChooserRegistrars>
-class StepChooser;
 // IWYU pragma: no_forward_declare db::DataBox
-template <typename StepChooserRegistrars>
-class StepChooser;
 /// \endcond
 
+/// The intended use for a step chooser.  This is used to control the
+/// classes via factories.
+namespace StepChooserUse {
+struct Slab;
+struct LtsStep;
+}  // namespace StepChooserUse
+
+/// \ingroup TimeSteppersGroup
+///
+/// StepChoosers suggest upper bounds on step sizes.  Concrete
+/// StepChoosers should define operator() returning the magnitude of
+/// the desired step (as a double).
+///
+/// Derived classes must indicate whether the chooser is usable as a
+/// step chooser, slab chooser, or both by inheriting from StepChooser
+/// with the appropriate `StepChooserUse` template argument. If it is
+/// usable as a step chooser (`StepChooserUse::LtsStep` or either,
+/// chosen by a template parameter), it must specify type aliases
```

Comment:
I don't understand what you mean with the part in parentheses, could you clarify?

Reply:
It's supposed to mean
```
template <typename Use>
class Derived : StepChooser<Use> {};
```
because you can't actually have a concrete class usable as either type.  I'll see about a clearer wording.

Reply:
Reads much better now, thanks!